trex.org

```diff
@@ -35,10 +35,16 @@ For sparse data structures such as electron replusion integrals,
 the data can be too large to fit in memory and the data needs to be
 fetched using multiple function calls to perform I/O on buffers.
 For more information on how to read/write sparse data structures, see
-the [[./examples.html][examples]].
+the [[./examples.html][examples]]. The ~sparse~ data representation implies the
+[[https://en.wikipedia.org/wiki/Sparse_matrix#Coordinate_list_(COO)][coordinate list]] representation, namely the user has to write a list
+of indices and values.
 
-For determinants, the ~special~ attribute is present in the type. This
-means that the source code is not produced by the generator, but hand-written.
+For the Configuration Interfaction (CI) and Configuration State Function (CSF)
+groups, the ~buffer~ data type is introduced, which allows similar incremental
```

Comment:
Shouldn't it be `buffered` instead of `buffer`? (below also)

Reply:
For me it's the same but I can change it if you want.

Reply:
Yes please!

Reply:
Done.